### PR TITLE
Reddit pipe

### PR DESCRIPTION
**Pull request by m13v**

### CODE REVIEW

File: examples/typescript/pipe-post-questions-on-reddit/pipe.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      "description": "Your OpenAI API key"
+    },
+    {
+      "name": "pageSize",
+      "type": "number",
+      "default": 100,
+      "description": "Number of records to retrieve from screenpipe per page for structured extraction, keep in mind LLMs have a context window limit. Increase this value if using audio."
+    },
+    {
+      "name": "additional prompt for logs",
```
Comment (louis030195) [inline examples/typescript/pipe-post-questions-on-reddit/pipe.json:58]: ??

File: examples/typescript/pipe-post-questions-on-reddit/pipe.json [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      "description": "GPT API URL"
+    },
+    {
+      "name": "gptModel",
+      "type": "string",
+      "default": "gpt-4o",
+      "description": "GPT Model"
+    },
+    {
+      "name": "openai_api_key",
```
Comment (louis030195) [inline examples/typescript/pipe-post-questions-on-reddit/pipe.json:46]: openaiApiKey (camelCase)

File: examples/typescript/pipe-post-questions-on-reddit/pipe.json (hunk 1 above)
Comment (louis030195) [reply examples/typescript/pipe-post-questions-on-reddit/pipe.json:58]: i think u meant customPrompt and summaryPrompt or something

**Comment by m13v** [comment]
fixed

**Pull request merged**